app/Http/Controllers/ResultsController.php

```diff
@@ -65,10 +65,10 @@ public function index(MismatchGetRequest $request, WikibaseAPIClient $wikidata):
         $props = array_merge(
             [
                 'user' => $user,
-                'item_ids' => $requestedItemIds,
+                'itemIds' => $requestedItemIds,
```

Comment:
Did eslint complain about a php file or was it a manual fix for consistency's sake?

Reply:
but did it lint this PHP file? I thought eslint only lints JS

Reply:
these are passed as Props to the Results page here: https://github.com/wmde/wikidata-mismatch-finder/pull/836/files#diff-f0e07e4099d24711f323a8ef5a958c7723c8bdaca46379665835500790de1c3fR245

If it doesnt have the same name some tests fail.

Reply:
Ah, I see. that sounds fine